repositories/like_repository.py

```
@@ -0,0 +1,61 @@
+import json
+import os
+from typing import List, Optional
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)